pkg/build/api/types.go

```diff
@@ -911,6 +915,9 @@ type GenericWebHookEvent struct {
 
 	// Env contains additional environment variables you want to pass into a builder container
 	Env []kapi.EnvVar
+
+	// StrategyOptions contains additional strategy-specific options for the build
+	StrategyOptions *BuildStrategyOptions
```

Comment:
I don't like the use of "options" - is there an example from elsewhere that had this?  I don't know that this makes sense nested, it's just extra abstraction.